### ISSUE TITLE
请问数据预处理中的标准化，是基于一方数据的还是多方数据的，如果是多方的是如何实现的

### ISSUE THREAD

**Issue by liujiachang** [secretflow/secretflow#1921]
### Question Category

Correctness/Accuracy

### Installation Source

Binary

### Secretflow Version

0.6.13

### Operating System and Distribution

20.04

### Python Version

3.10

### Question Details

```shell
我正在研究联邦学习中数据预处理的部分操作，想了解这部分内容是如何实现的。
```

### Reproduction Code

```shell
from secretflow.preprocessing import MinMaxScaler

scaler = MinMaxScaler()

scaled_sepal_len = scaler.fit_transform(df['sepal length (cm)'])

print('Min: ', scaled_sepal_len.min())
print('Max: ', scaled_sepal_len.max())
```

### Additional Context or Logs

```shell

```
